examples/nonlocal_games/README.txt

```diff
@@ -0,0 +1,4 @@
+Nonlocal Games
```

Comment:
It might be fine to remove this file. When the examples are added to the gallery, we can use tags to categorize them.

https://sphinx-tags.readthedocs.io/en/latest/

@vprusso Thoughts?

Reply:
@vprusso Pinging you in again because I think you missed this comment. Do we want to separate tutorials similar to what's in this PR or have all tutorials on one page with `sphinx-tags`?

With the former, we have to think about maintaining multiple index pages. 
https://toqito--1175.org.readthedocs.build/en/1175/auto_examples/index.html
https://toqito--1175.org.readthedocs.build/en/1175/auto_examples/nonlocal_games/index.html

The latter is done automatically. See an example in mitiq https://mitiq.readthedocs.io/en/stable/examples/tags.html